src/utils/debugging/log.ts

```diff
@@ -0,0 +1,6 @@
+type Log = <T>(...args:Array<T>) => T
+
+export const log:Log = (...args) => {
+    console.log(...args)
+    return args[args.length - 1]
```

Comment:
is this function supposed to have a `return` value?

Reply:
nevermind, i can see further below that this is written as intended. Though it's a bit odd to me why a log function would return something, and that that something is the last argument passed in... but forgetting about that for a moment, it would be good to handle the case where no args are passed

Reply:
chatted on this, shoudl be good!